ci-operator/step-registry/upi/conf/libvirt/network/upi-conf-libvirt-network-ref.yaml

```diff
@@ -2,6 +2,10 @@ ref:
   as: upi-conf-libvirt-network
   from: libvirt-installer
   commands: upi-conf-libvirt-network-commands.sh
+  env:
+  - name: INSTALLER_TYPE
+    documentation: The type of installer to use for deployment. Supported values are [agent, assisted, default]
```

Comment:
Do we currently support `assisted` in the upi workflows?  We should probably only include the options that are fully supported if it isn't.

Reply:
We do not support assissted yet - this is a good note. I completely forgot about this line.